src/robots/ev3/ev3.cc

```diff
@@ -26,8 +26,8 @@ checkMotor(const EV3::MotorType &motor, const std::string &label)
     }
 }
 
-EV3::EV3(const ev3dev::address_type leftMotorPort,
-         const ev3dev::address_type rightMotorPort)
+EV3::EV3(const ev3dev::address_type &leftMotorPort,
```

Comment:
aren't these literally pointers or some other small type that probably shouldn't be passed by reference? (they probably should not have been ``const`` if that's true but hey)

Reply:
These are actually just ``typedef``'d ``std::string``s (though why they bothered with that I don't know), so in this particular case it does make sense.